package.json

```diff
@@ -4,45 +4,47 @@
   "scripts": {
     "ng": "ng",
     "build": "ng build --prod",
-    "test": "ng test --no-progress --no-watch",
+    "test": "ng test --no-progress --no-watch --code-coverage",
     "lint": "ng lint",
     "release": "cd projects/ngx-speculoos && standard-version --infile ../../CHANGELOG.md",
-    "doc": "node scripts/prepare-doc.js && cd projects/ngx-speculoos && compodoc"
+    "doc": "node scripts/prepare-doc.js && cd projects/ngx-speculoos && compodoc",
+    "codecov": "codecov"
   },
   "private": true,
   "devDependencies": {
+    "@angular-devkit/build-angular": "~0.6.3",
```

Comment:
`yarn add` reordered the keys...

Reply:
Yeah, it's nice, because the order is alphabetical, but it's a pain, because angular deps are mixed with cli dependencies, which don't have the exact same version number.